ssg/utils.py

```diff
@@ -40,6 +43,38 @@ def map_name(version):
                        % (version))
 
 
+def to_name(prod):
```

Comment:
One thought is that we could move this to a new file, `ssg/jinja-macros.py`. Note that we can't actually add these to `products.py` since that'd create a cyclic dependency: `ssg/jinja.py -> ssg/products.py -> ssg/yaml.py -> ssg/jinja.py`

Reply:
@cipherboy Could it have a better name? For example `prodtype_to_name`?

Reply:
@jan-cerny Sure, I'll fix this today if I get time.

Reply:
Renamed function to `prodtype_to_name`.